tests/torch/fx/test_quantizer_config.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from nncf.quantization.algorithms.min_max.torch_fx_backend import FXMinMaxAlgoBackend
+from tests.cross_fw.test_templates.models import NNCFGraphToTest
+from tests.cross_fw.test_templates.models import NNCFGraphToTestDepthwiseConv
+from tests.cross_fw.test_templates.models import NNCFGraphToTestSumAggregation
+from tests.cross_fw.test_templates.test_quantizer_config import TemplateTestQuantizerConfig
+from tests.torch.ptq.helpers import get_depthwise_conv_nncf_graph
+from tests.torch.ptq.helpers import get_single_conv_nncf_graph
+from tests.torch.ptq.helpers import get_sum_aggregation_nncf_graph
+
+
+class TestQuantizerConfig(TemplateTestQuantizerConfig):
+    def get_algo_backend(self):
+        return FXMinMaxAlgoBackend()
+
+    @pytest.fixture
+    def single_conv_nncf_graph(self) -> NNCFGraphToTest:
+        return get_single_conv_nncf_graph()
```

Comment:
Please do not use pytorch nncf grpahs, please create the same functions in torchFX test helpers as it done here https://github.com/openvinotoolkit/nncf/blob/develop/tests/torch/ptq/helpers.py but do not specify the layer attributes as TorchFX nncf graph does not require layer attributes

Reply:
> Please do not use pytorch nncf grpahs, please create the same functions in torchFX test helpers as it done here https://github.com/openvinotoolkit/nncf/blob/develop/tests/torch/ptq/helpers.py but do not specify the layer attributes as TorchFX nncf graph does not require layer attributes

Alright yes. Thank you for your guidance.